types/schema.d.ts

```diff
@@ -14,7 +14,7 @@ export type Project = {
   assembly_district: number;
   project_image: string | null;
   additional_information: string | null;
-  key_development_milestones: object | null;
+  key_development_milestones: [Milestone] | null;
```

Comment:
Aren't arrays of items declared as `Milestone[]`? I may be getting confused with java lol